dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -218,8 +218,10 @@
     @RequestMapping( method = RequestMethod.GET )
     public @ResponseBody RootNode getObjectList(
         @RequestParam Map<String, String> rpParameters, OrderParams orderParams,
-        HttpServletResponse response, User currentUser ) throws QueryParserException
+        HttpServletResponse response) throws QueryParserException
     {
+        User currentUser = currentUserService.getCurrentUser();
```

Comment:
Is the parameter injection of `User` not supported anymore?

Reply:
Yes, I tried to figure out this but really could not find where this is configured. Do you have any idea about where this is defined?

Reply:
You can find this under the `mvc` package in dhis-web-api, it also has argument resolvers for DhisApiVersion etc.

`CurrentUserHandlerMethodArgumentResolver`